packages/icons/CHANGELOG.md

```
@@ -2,6 +2,16 @@
 
 Nimbus-icons is an open source icon system built by Nuvemshop / Tiendanube.
 
+## 2025-08-22 `1.13.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)